StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -243,6 +278,16 @@ extension PaymentSheetVerticalViewController: VerticalPaymentMethodListViewContr
             // TODO Set payment option
             return true
         case let .new(paymentMethodType: paymentMethodType):
+            // Update the header view, hide wallet if needed and show header label if needed
+            walletHeaderView?.isHidden = true
+            headerView.isHidden = false
+            if paymentMethodType == .stripe(.card) {
+                let text = savedPaymentMethods.isEmpty ? String.Localized.add_card : String.Localized.add_new_card
+                headerView.set(text: text)
+            } else {
+                headerView.update(with: paymentMethodType)
+            }
```

Comment:
Hmm this doesn't animate but it should somehow, it looks too choppy. Not sure what the best approach is. Could maybe just add an animated flag, but I'm not sure it's still going to position itself in a good way.  Might look better if the header is on the form?

Reply:
Hmm, are we expecting an animation like a fade or something similar?